src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -4832,15 +4832,7 @@ int RGWRados::copy_obj(RGWObjectCtx& obj_ctx,
   RGWRados::Object dest_op_target(this, dest_bucket_info, obj_ctx, dest_obj);
   RGWRados::Object::Write write_op(&dest_op_target);
 
-  string tag;
-
-  if (ptag) {
-    tag = *ptag;
-  }
-
-  if (tag.empty()) {
-    append_rand_alpha(cct, tag, tag, 32);
```

Comment:
i guess that these tags are orphans anyway?
so, we are not creating a bigger issue when we will migrate RGW and change the way we generat the tags?

Reply:
the tag is only used for the ref_count, it is not set as id-tag (there is another code setting it regardless of the ptag value)

Reply:
BTW, not all flows pass in a valid tag, some are passing in a NULL value and then a random-alpha is generated